components/Proposals/MintConditionalTokenCard.tsx

```diff
@@ -39,29 +39,29 @@ export function MintConditionalTokenCard() {
   const [isMinting, setIsMinting] = useState(false);
   if (!markets) return null;
 
-  const { metaToken, usdcToken } = useConditionalTokens();
-
-  const [token, setToken] = useState<Balance | undefined>();
+  const { baseToken, quoteToken } = useConditionalTokens();
+  const [token, setToken] = useState<Balance | undefined>(baseToken);
 
   useEffect(() => {
     setToken((prev) => {
-      if (!prev) return metaToken;
-      return prev.symbol === 'META' ? metaToken : usdcToken;
+      if (!prev) return baseToken;
+      return prev.symbol === baseToken?.symbol ? baseToken : quoteToken;
     });
-  }, [metaToken, usdcToken]);
+    // not proud of this, TODO clean this up
```

Comment:
@R-K-H lmk if you want to prioritize cleaning this up. There are some special hooks we could look at implementing instead of this heck. It does work well though, but just doesn't look great.